test/projects/sorting/heapsort.py

```diff
@@ -0,0 +1,40 @@
+"""Run heapsort on a list a
+    >>> a = [32,46,77,4344564,7322,3,46,7,32457,7542,4,667,54,]
+    >>> heapsort(a)
```

Comment:
Sorry to be slightly picky, but should this be an inplace sort? 

If we take a look at the python stdlib

* a.sort() -> sorts inplace
* sorted(a) -> returns new list (new object)

Reply:
Since all we care about is the output, I don't think it matters if we perform an in place sort or not. That said, heapsort isn't documented yet, so it's hard to say.